brainscore_vision/model_helpers/activations/temporal/inputs/video.py

```diff
@@ -0,0 +1,147 @@
+import cv2
+from decord import VideoReader
+import numpy as np
+from PIL import Image as PILImage
+import multiprocessing as mp
+
+from .base import Stimulus
+from .image import Image
+from brainscore_vision.model_helpers.activations.temporal.utils import batch_2d_resize
+
+
+def get_video_stats(video_path):
+    cap = cv2.VideoCapture(video_path)
+    length = int(cap.get(cv2.CAP_PROP_FRAME_COUNT))
+    width  = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+    height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+    fps    = cap.get(cv2.CAP_PROP_FPS)
+    size = (width, height)
+    duration = length / fps * 1000
+    cap.release()
+    return fps, duration, size
+
+
+def get_image_stats(image_path):
+    img = PILImage.open(image_path)
+    size = img.size
+    img.close()
```

Comment:
```suggestion
    with PILImage.open(image_path) as img:
        return img.size
```
use context managers